jinja2/bccache.py

```diff
@@ -3,9 +3,8 @@
     jinja2.bccache
     ~~~~~~~~~~~~~~
 
-    This module implements the bytecode cache system Jinja is optionally
-    using.  This is useful if you have very complex template situations and
-    the compilation of all those templates slow down your application too
+    This module implements the bytecode cache system that Jinja optionally uses.  This is useful if you have very complex template situations and
```

Comment:
This line is too long. Could you please break this up to respect the max line width here?

Reply:
Sorry I missed this. Fixed!